.gitignore

```diff
@@ -1 +1,3 @@
 /target
+/gb_testing
+/.vscode
```

Comment:
I would like to avoid editor-specific ignores in the upstream repo.

You can add entries local to your Git repo in `.git/info/exclude` ([documentation](//git-scm.org/docs/gitignore)).

Reply:
Excellent, that has been fixed.